auth-server/docker-compose.yaml.example

```diff
@@ -1,5 +1,10 @@
 version: "3.6"
 services:
+  redis:
+    image: redis
+    command: redis-server --requirepass redis
```

Comment:
Can we pass this as env, we'll need it I guess. Also just read about the 2nd point from https://redis.io/topics/quickstart#securing-redis
> Use a configuration file where the bind directive is set in order to guarantee that Redis listens on only the network interfaces you are using. For example only the loopback interface (127.0.0.1) if you are accessing Redis just locally from the same computer, and so forth.

Not sure how to do that. It's probably a todo for our redis server and we shouldn't bother too much for now if we move away from docker.

Reply:
This is only needed when redis machine is publicly accessible. In our case it will be internal like node watcher